lib/JiraClient.ts

```diff
@@ -9,6 +9,25 @@ export class JiraClient {
         this.token = Buffer.from(`${user}:${token}`).toString('base64');
     }
 
+    public async createIssue(projectKey: string, issueType: any, summary: string): Promise<string> {
```

Comment:
Based on the other comment (below or above)
```suggestion
    public async createIssue(projectKey: string, issueTypeId: number, summary: string): Promise<string> {
```